server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+
+After registering their WebAuthn authenticator, the user carries out the following operations :
+
+- open the login form
+- fill in the items and push `Login` button
+- the list of registered Webauthn Authenticators' labels is shown. push `Authenticate` button as it is
+- the user's browser asks the user to authenticate by their WebAuthn authenticator
+
+==== View Registered WebAuthn Authenticator
+
+An administrator and a user can view the following information :
+
+- https://www.w3.org/TR/webauthn/#credential-id[Credential ID]
+- Label (WebAuthn authenticator's label the user entered on registering it)
+- https://fidoalliance.org/specs/fido-v2.0-rd-20180702/fido-metadata-statement-v2.0-rd-20180702.html#authenticator-attestation-guid-aaguid-typedef[AAGUID]
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Users -> (user) -> Credentials` tab
+- view `Manage WebAuthn Authenticator` area
+
+A user carries out the following operations on <<_account-service, `User Account Service`>>  :
+
+- view `Account` page
+
+==== Edit Registered WebAuthn Authenticator
+
+A user can edit the following information :
+
+- Label (WebAuthn authenticator's label the user entered on registering it)
+
+A user carries out the following operations on <<_account-service, `User Account Service`>>  :
+
+- view `Account` page
+- edit the text in `Public Key Credential Label`
+- push `Save` button
+
+==== Delete Registered WebAuthn Authenticator
+
+An administrator can delete the users' registered WebAuthn authenticators.
+
+An Administrator carries out the following operations on `Admin Console` :
+
+- open `Users -> (user) -> Credentials` tab
+- on `Disable Credentials` area, add `webauthn` to `Disable Types`
+- push `Disable Credential Types` button
+
+==== Re-Register WebAuthn Authenticator
+
+A user can re-register their WebAuthn authenticator. Newly registered WebAuthn authenticator overrides the old registered one.
+
+At first, an administrator carries out the following operations on `Admin Console` to require the user to register their WebAuthn authenticator after their login :
+
+- open `Users -> (user) -> Details` tab
+- set `WebAuthn Register` on `Required User Actions`
+- push `Save` button
+
+After that, the user carries out the following operations :
+
+- open the login form
+- fill in items and push `Login` button
+- after logged in, the user's browser asks the user to register their WebAuthn authenticator
+- after successful registration, the user's browser asks the user to enter the text as their just registered WebAuthn authenticator's label
+
+==== Configuration
+
+An administrator can configure WebAuthn related operations as `WebAuthn Policy` per realm.
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Authentication -> WebAuthn Policy` tab
+- configure items and push `Save` button
+
+The configurable items and their description are shown just below.
```

Comment:
instead of "are shown just below" please use simply "follow."

Reply:
Thank you. I'll fix this sentence as you suggested.